### ISSUE TITLE
Typing effect stopped working after package upgrade

### ISSUE THREAD

**Issue by singhshashi** [adamjking3/react-typing-animation#11]
I upgraded a bunch of my node modules and this one was updated to the latest version 1.2.1. 

But after upgrade, the Typing effect stopped working. The showAddTask is not fired neither does the firstIntroInstruction shown. 
```

 <Typing onFinishedTyping={this.showAddTask} speed={25}>
          <span>{firstIntroInstruction}</span>
          <Typing.Reset count={1} delay={500} />
          <span>{secondIntroInstruction}</span>
        </Typing>
```

**Comment by adamjking3** [comment]
Looking into this.

**Comment by adamjking3** [comment]
This should be fixed in v1.3.0.
